config/zones.json

```diff
@@ -3337,6 +3337,35 @@
     },
     "timezone": "Europe/Riga"
   },
+  "MA": {
+    "bounding_box": [
```

Comment:
@veqtrus can you update this to:
```
    "bounding_box": [
      [
        -17.51374332534519,
        20.919971097581723
      ],
      [
        -0.5319994709999492,
        36.426519149000015
      ]
    ],
```

You can create bounding boxes for new or existing zones in `config/zones.json` by running the following from the `web` folder:
1) Run: `docker-compose run --rm web ./topogen.sh`
2) Update the zone you want to update in `config/zones.json` with `"bounding_box": null`
3) Run: `node generate-zone-bounding-boxes.js`

Reply:
Done